src/components/AssetModels/AssetModelsComponent.jsx

```diff
@@ -86,20 +92,27 @@ export class AssetModelsComponent extends React.Component {
 
             <Table.Footer>
               <Table.Row>
-                <Table.HeaderCell colSpan="4">
+                <Table.HeaderCell colSpan="5" id="pagination-header">
```

Comment:
Same previous comment applies here

Reply:
Please move `<Table.HeaderCell...` into `{!_.isEmpty(this.props.assetModels) && (...)` as you have done in previous component. I.e.
```
{!_.isEmpty(this.props.assetModels) && (
  <Table.HeaderCell colSpan="5" id="pagination-header">
    ...
  </Table.HeaderCell>
)}